configure.ml

```diff
@@ -22,8 +22,10 @@ let distributed_exec = ["coqtop";"coqc";"coqchk";"coqdoc";"coqworkmgr";
 let verbose = ref false (* for debugging this script *)
 
 (** * Utility functions *)
-
-let die msg = eprintf "%s\nConfiguration script failed!\n" msg; exit 1
+let cfprintf oc = kfprintf (fun oc -> fprintf oc "@\n%!") oc
```

Comment:
"@\n" syntax is Format-only. Either use `Format` of switch to regular "\n"

Reply:
Fixed, thanks!